static/css/bundle.css

```
@@ -2671,6 +2671,457 @@ body.ldj .welcome > .form.form-settings > .welcome-catalog > .settings-container
+    margin: 0px 0px 24px 0px;
+}
+
+.components-section-lede > code {
+    background-color: rgba(31, 35, 48, 0.06);
+    border-radius: 4px 4px 4px 4px;
+    -o-border-radius: 4px 4px 4px 4px;
+    -ms-border-radius: 4px 4px 4px 4px;
+    -moz-border-radius: 4px 4px 4px 4px;
+    -khtml-border-radius: 4px 4px 4px 4px;
+    -webkit-border-radius: 4px 4px 4px 4px;
+    font-family: "SFMono-Regular", "Menlo", "Monaco", "Consolas", monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)